scss/mixins/_color-mode.scss

```diff
@@ -13,9 +13,18 @@
       }
     }
   } @else {
-    [data-bs-theme="#{$mode}"] {
-      @content;
+    // Boosted mod
+    @if $mode == "light" {
```

Comment:
Should we introduce the concept of "default theme" that could be useful for users?
Or maybe use the parameter `$root` ? Because it works for us but maybe not for every use case